grammars/javascript.cson

```diff
@@ -1185,25 +1175,94 @@
   'arguments':
     'patterns': [
       {
-        'include': '$self'
+        'begin': '\\('
+        'beginCaptures':
+          '0':
+            'name': 'punctuation.definition.arguments.begin.js'
+        'end': '\\)'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.definition.arguments.end.js'
+        'name': 'meta.arguments.js'
+        'patterns': [
+          {
+            'include': '$self'
+          }
+        ]
       }
     ]
-  'method_call':
+  'method_calls':
     'patterns': [
       {
-        'comment': '.methodCall(arg1, "arg2", [...])'
-        'begin': '(\\.)([a-zA-Z_$][\\w$]*)\\s*(\\()'
+        # .methodCall(arg1, "arg2", [...])
+        'begin': '(\\.)([\\w$]+)\\s*(?=\\()'
```

Comment:
It allows to match arguments (currently not tagged) even if function name is illegal:
![](http://g.recordit.co/nHOed6pgSj.gif)